sv-parser/build.sh

```diff
@@ -10,6 +10,7 @@ fi
 curl https://sh.rustup.rs -sSf | sh -s -- -y
 source $HOME/.cargo/env
 
+git checkout $GIT_DESCRIBE_TAG
```

Comment:
This line doesn't really make any sense. The repository will already be checked out at this tag?

Reply:
Really? In my understanding, HEAD in master branch is checked out because `meta.yml` is below:

```
source:
  git_url: https://github.com/dalance/sv-parser
  git_rev: master
```

I tried the following,  but it caused checkout error.

```
source:
  git_url: https://github.com/dalance/sv-parser
  git_rev: {{ GIT_DESCRIBE_TAG }}
```